src/systems/README.md

```diff
@@ -10,7 +10,7 @@ Follow the steps below if you want to build the content of this folder without u
 
 1. Install the `wasm32-wasi` target by running: `rustup target add wasm32-wasi`.
 2. Install LLVM 11 which can be found [here](https://releases.llvm.org/download.html), ensure to add it to your PATH.
-3. Run `cargo build --target wasm32-wasi` in the console at the top-level of the a32nx repository.
+3. Run `cargo build --target wasm32-wasi` in the console at the top-level of the a32nx repository. Make sure to have downloaded the `Samples` folder of the SDK (needed to be downloaded separately from the core) in order to build `msfs-rs`.
```

Comment:
> 3. ... Make sure to have downloaded the `Samples` folder of the SDK (needed to be downloaded separately from the core) in order to build `msfs-rs`.

Change to

You must have the SDK installed and have the 'Samples' folder of the SDK downloaded (this must be acquired seperately from the core) in order to build `msfs-rs`.